daemon/create.go

```diff
@@ -108,6 +111,11 @@ func (daemon *Daemon) containerCreate(ctx context.Context, daemonCfg *configStor
 	if opts.params.HostConfig == nil {
 		opts.params.HostConfig = &containertypes.HostConfig{}
 	}
+	if opts.params.TransformBinds {
+		if err := transformBinds(opts.params.HostConfig); err != nil {
```

Comment:
I don't see why we are going to all this trouble here when we are already going to be parsing these mounts elsewhere, we can tell the it to set rro as a default based on the new param.

Reply:
You're right, fixed!